src/platform/plugins/shared/discover/public/embeddable/types.ts

```diff
@@ -64,6 +64,7 @@ export type SearchEmbeddableSerializedAttributes = Omit<
 // These are options that are not persisted in the saved object, but can be used by solutions
 // when utilising the SavedSearchComponent package outside of dashboard contexts.
 export interface NonPersistedDisplayOptions {
+  solutionNavIdOverride?: string;
```

Comment:
Can we add the expected values to facilitate autocomplete? Like the following:

```suggestion
  solutionNavIdOverride?: 'oblt' | string;
```

Reply:
Good idea and yes we can. Updated here: https://github.com/elastic/kibana/pull/205986/commits/a6850d69e613936eaea3d33326906e05c5be6fc6.